src/models/capability/Thermostat.js

```diff
@@ -158,6 +158,9 @@ export default {
   HVACMODE_COOL: 'COOL',
   HVACMODE_HEAT: 'HEAT',
   HVACMODE_ECO: 'ECO',
+  // TODO: Add these 2 properties below to the capability file in arcusplatform
```

Comment:
The pattern for this appears to be using an enum. I'm not sure why this is the case for fanmode - the action item here is probably to convert it to an enum, otherwise the capability auto-generator will need to be updated (a ticket can be raised under arcusplatform this)

Reply:
https://github.com/arcus-smart-home/arcusplatform/issues/69
Opened issue as requested.